verification/tests/endpoints/contracts/storage/metadata.json

```diff
@@ -0,0 +1,88 @@
+{
```

Comment:
suggest new folder name: `storage` -> `test_data`?

Reply:
I suggest moving everything out of `endpoints` directory. So that we will have `verification/tests/contracts/contract_name/..` with all contracts and `verification/tests/sourcify.rs` with actual test file.

Also, in integration tests we do not need `mod.rs` to combine them, as each `.rs` file is a separate crate by itself which runs all tests.